Automattic/newspack-newsletters
# Remove Template Selection UI

### All Submissions:






### Changes proposed in this Pull Request:

Remove the Mailchimp Template selection UI and related APIs.

### How to test the changes in this Pull Request:

Verify the Template dropdown is no longer in the sidebar. Verify all else works normally.

### Other information:
